bin/check_google_doc_export.rb

```diff
@@ -16,12 +16,30 @@
 require_relative '../lib/article_json'
 
 doc_id = '1E4lncZE2jDkbE34eDyYQmXKA9O26BHUiwguz4S9qyE8'
-url = "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
+url =
+  "https://docs.google.com/feeds/download/documents/export/Export?id=#{doc_id}&exportFormat=html"
 expected_document = JSON.parse(File.read('spec/fixtures/reference_document_parsed.json'))
 
 exported_doc = Net::HTTP.get(URI.parse(url))
 document = JSON.parse(ArticleJSON::Article.from_google_doc_html(exported_doc).to_json)
 
+# `source_url` (for hosted images) is dynamic, so we need to remove it from the comparison`
+def nullify_source_url(hash)
+  content = hash['content']
+  return unless content.is_a?(Array)
+
+  content.each do |key, value|
```

Comment:
When the `content` is an array, then the `value` is always `nil`, isn't it?

Reply:
Amended, thanks.